exotica_python/test/test_no_unknown_initializer_types.py

```diff
@@ -1,8 +1,15 @@
 from __future__ import print_function
 import subprocess
+import os
 
-output = subprocess.check_output("python -c 'import pyexotica as exo ; initializers = exo.Setup.get_initializers()'", shell=True)
+output = None
+if os.environ['ROS_PYTHON_VERSION'] == '2':
+    output = subprocess.check_output("python -c 'import pyexotica as exo ; initializers = exo.Setup.get_initializers()'", shell=True)
+elif os.environ['ROS_PYTHON_VERSION'] == '3':
+    output = subprocess.check_output("python3 -c 'import pyexotica as exo ; initializers = exo.Setup.get_initializers()'", shell=True)
+else:
+    assert False
 
-if "Skipping" in output:
+if b"Skipping" in output:
     print(output)
-    assert "Skipping" not in output
+    assert b"Skipping" not in output
```

Comment:
Codacy found an issue: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/ipab-slmc/exotica/pullRequest?prid=5739305)